From 9dff961d157f0f8ec82de9eb6ab07a212ffbde50 Mon Sep 17 00:00:00 2001
From: hyphop <art@khadas.com>
Date: Thu, 7 May 2020 22:53:17 +0900
Subject: [PATCH 24/38] vim2_spifc_fix_freq_and_id

change chip name for spi and  max freq to 104Mhz
---
 dts/upstream/src/arm64/amlogic/meson-gxm-khadas-vim2.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dts/upstream/src/arm64/amlogic/meson-gxm-khadas-vim2.dts b/dts/upstream/src/arm64/amlogic/meson-gxm-khadas-vim2.dts
index 07e7c3bede..d68b747d5e 100644
--- a/dts/upstream/src/arm64/amlogic/meson-gxm-khadas-vim2.dts
+++ b/dts/upstream/src/arm64/amlogic/meson-gxm-khadas-vim2.dts
@@ -376,7 +376,7 @@
 	w25q32: flash@0 {
 		#address-cells = <1>;
 		#size-cells = <1>;
-		compatible = "winbond,w25q16", "jedec,spi-nor";
+		compatible = "winbond,w25q128fw", "jedec,spi-nor";
 		reg = <0>;
 		spi-max-frequency = <104000000>;
 	};
-- 
2.17.1

